docs/API.md

```diff
@@ -784,6 +784,8 @@ Response:
 }
 ```
 
+> **Note:** The `status` field in the JSON response does not indicate the status of the device itself, but rather the status of the HTTP response. To get an accurate representation of whether the device state has settled, use the `appState` field. `appState` will be one of `applying` or `applied`.
```

Comment:
Maybe this should go before the example, it will make it more likely than people read it.

Reply:
Sure thing.